ServerRecording/src/main/java/com/acsrecording/api/Controller/CallRecordingController.java

```diff
@@ -50,22 +56,44 @@
 public class CallRecordingController  {
     Map<String,String> recordingDataMap;
     String container;
+    String ACSAcquiredPhoneNumber;
     String blobStorageConnectionString;
+    String BaseUri;
     Logger logger;
     String recordingFileFormat;
     private final com.azure.communication.callautomation.CallAutomationClient callAutomationClient;
 
     CallRecordingController() {
         ConfigurationManager configurationManager = ConfigurationManager.getInstance();
         String connectionString = configurationManager.getAppSettings("Connectionstring");
+        ACSAcquiredPhoneNumber= configurationManager.getAppSettings("ACSAcquiredPhoneNumber");
         container = configurationManager.getAppSettings("ContainerName");
         blobStorageConnectionString = configurationManager.getAppSettings("BlobStorageConnectionString");
```

Comment:
remove all storage related codes.

Reply:
Created new folder as CallRecoeding For ServerRecording one..